Sources/Slacket/Kitura/Middleware/HealthCheckMiddleware.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  HealthCheckMiddleware.swift
+//  Slacket
+//
+//  Created by Jeremi Kaczmarczyk on 22.06.2016.
+//
+//
+
+import Foundation
+
+import Kitura
+import HeliumLogger
+import LoggerAPI
+
+class HealthCheckMiddleware: RouterMiddleware {
+    
+    func handle(request: RouterRequest, response: RouterResponse, next: () -> Void) throws {
+        try response.end()
```

Comment:
Currently the default status is `.unknown` and if it is not changed manually, then it is set to `.ok` by default.